### ISSUE TITLE
Read/unread states not saved between different accounts

### ISSUE THREAD

**Issue by ivkowalenko** [spazproject/spaz-desktop-titanium#10]
When I switch between accounts on Spaz, the read/unread states of messages are not saved. Ordinarily this isn't a big issue, except that it triggers the "new messages" sound and displays all the notifications for each message loaded.

System run on: Mac OS X 10.6.7, 4 GB RAM, Intel Core 2 Duo at 2.6 GHz
Spaz Version: 1.0 (it says in the about dialog at least)

**Comment by funkatron** [comment]
Is that different than the AIR app?

**Comment by ivkowalenko** [comment]
Yes. The AIR app (v0.9.21) remembers which tweets I have "marked read" when I'm switching between accounts.

**Comment by funkatron** [comment]
This needs to be rewritten to work with webkitSQL or another DB system supported in Titanium.
